src/main/antlr4/de/zalando/beard/BeardParser.g4

```diff
@@ -10,15 +10,13 @@ import de.zalando.beard.ast.*;
 
 beard
 locals [scala.collection.immutable.List<Statement> result]
-      : (extendsStatement NL?)? (NL* contentForStatement)* statement*
```

Comment:
This change makes the grammar simples but also less strict.

It could be possible to have "extends" in the middle of a file for example. I think that if we want to prevent that, we should add checks to the AST and not the add this to the grammar.

Making the grammar more homogeneous and simple makes it easy to understand, while adding rules to the AST, or even transforming the AST at the end is easier to understand and maintain.

Wdyt?